applications/nrf5340_audio/README.rst

```diff
@@ -1227,6 +1230,34 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 
 .. _nrf53_audio_app_porting_guide:
 
+Testing FOTA upgrades
+=====================
+
+`nRF Connect Device Manager`_ can be used for testing FOTA upgrades.
+Complete the following steps to test upgrading the firmware on both cores at the same time on a gateway device (the procedure is identical for upgrading headset firmware):
+
+1. Install `nRF Connect Device Manager`_ on your Android or iOS device.
```

Comment:
An Android/iOS device is also a requirement.

Reply:
Added a Requirements for FOTA section